spring-session-docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -1,4 +1 @@
 = What's New
-
-Check also the Spring Session BOM https://github.com/spring-projects/spring-session-bom/wiki#release-notes[release notes]
```

Comment:
3.0 needs _What's New_ updated, but in the meantime we should probably remove this reference to external BOM repo.